src/mon/NVMeofGwTypes.h

```diff
@@ -226,6 +229,13 @@ struct NvmeGwTimerState {
   NvmeGwTimerState() {};
 };
 
+typedef struct Gw_Epoch {
+    epoch_t epoch;
+    Gw_Epoch(epoch_t epoch) : epoch(epoch){
+    };
+    Gw_Epoch():Gw_Epoch(0) {};
+}GwEpoch;
```

Comment:
Why?  It seems like it would be a lot simpler to just use epoch_t -- ceph developers already know that type.  Defining a new one here merely means that any reader must go find it to figure out what the type actually does.

Reply:
ok